client/components/jetpack/card/npip/jetpack-free-card-npip/index.tsx

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
Recreated from i5 specifically to accommodate styling and slight markup differences re: feature list bullet points.